tstyche.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "testFileMatch": ["**/__typetests__/*.test.ts", "**/__typetests__/*.test.tsx"]
```

Comment:
Hm.. The default `"**/__typetests__/*.test.*"` pattern did not work? Sounds you don’t need `tstyche.config.json`.

Reply:
Yes your right